src/som/interpreter/actors/Actor.java

```diff
@@ -252,8 +262,8 @@ void doRun() {
 
       t.currentlyExecutingActor = actor;
 
-      if (VmSettings.ACTOR_TRACING) {
-        ActorExecutionTrace.recordActivityContext((TracingActor) actor, tracer);
```

Comment:
Was this cast to `TracingActor` unnecessary in the first place? Otherwise I didn't catch a change that would explain why it is not needed anymore

Reply:
The tracer nodes were changed, they are now specialising on the activity type and are no longer restricted to actors. of course this is not visible here. I probably should refactor ActorExecution trace, the name isn't accurate anymore